.gitpod.yml

```diff
@@ -0,0 +1,115 @@
+## Learn more about this file at 'https://www.gitpod.io/docs/references/gitpod-yml'
```

Comment:
Don't know what to do with these comments tbh.

They're giving some extra context, which is a good thing imo.
But it's also making the file more verbose I think.

If we decide to keep the comments, I think we should add them in the other stacks as well, so we keep this file the same on all our stacks.